### ISSUE TITLE
Unclear explanation of 4 epx base unit

### ISSUE THREAD

**Issue by JVimes** [MicrosoftDocs/windows-uwp#2839]
The sentence I bolded here seems unclear to me:

> UWP scales across a range of devices with scaling plateaus of 100%, 125%, 150%, 175%, 200%, 225%, 250%, 300%, 350%, and 400%. **The base unit is 4 because it's the only integer that can be scaled by non-whole numbers (for example, 4*1.5 = 6).**

This seems wrong because all integers can be scaled by non-whole numbers. How about rewording it to:

> UWP scales across a range of devices with scaling plateaus of 100%, 125%, 150%, 175%, 200%, 225%, 250%, 300%, 350%, and 400%. **The base unit is 4 because it can be scaled to these plateaus as a whole number (for example, 4*1.5 = 6).**

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 05005ba0-1609-f4e1-16e2-47be162453e8
* Version Independent ID: 61bf242a-12ad-606f-e125-83ed6cb72fd7
* Content: [Introduction to Windows app design (Windows apps) - UWP applications](https://docs.microsoft.com/en-us/windows/uwp/design/basics/design-and-ui-intro)
* Content Source: [windows-apps-src/design/basics/design-and-ui-intro.md](https://github.com/MicrosoftDocs/windows-uwp/blob/docs/windows-apps-src/design/basics/design-and-ui-intro.md)
* Product: **uwp**
* Technology: **design**
* GitHub Login: @jwmsft
* Microsoft Alias: **jimwalk**

**Comment by jwmsft** [comment]
Thank you for your more clear and accurate suggestion. The page has been updated.
